src/dat.c

```diff
@@ -228,7 +229,7 @@ int eraDat(int iy, int im, int id, double fd, double *deltat )
    if (i < 0) return -5;
 
 /* Get the Delta(AT). */
-   da = changes[(i < 0) ? 0 : i].delat;
+   da = changes[i].delat;
```

Comment:
Ok, sounds good.  If there ends up being no local change necessary at all, then you can just remove the section about that change from the README entirely.